docs/BOOT_LOADER_INTERFACE.md

```diff
@@ -58,6 +58,17 @@ variables. All EFI variables use the vendor UUID
 * The EFI variable `LoaderEntryDefault` contains the default boot loader entry
   to use. It contains a NUL-terminated boot loader entry identifier.
 
+* The EFI variable `LoaderEntrySysFail` contains the sysfail boot loader entry
+  to use. Sysfail boot entries can be used optional tweaking the automatic
+  selection order in case a failure state of the system in some form is detected.
```

Comment:
It sounds ambiguous. First of all, failure here means something in boot sequence?
Please declare more explicitly. And, may be better to add some example failure cases.

Reply:
I've completely rephrased the whole description